grafana-plugin/src/plugin.json

```
@@ -186,6 +186,465 @@
       ]
     }
   ],
+  "roles": [
+    {
+      "role": {
+        "name": "plugins.app:grafana-oncall-app:admin-user-settings-admin",
+        "displayName": "",
+        "description": "",
+        "group": "OnCall",
+        "permissions": [
+          { "action": "grafana-oncall-app.integrations:write" },
+          { "action": "grafana-oncall-app.escalation-chains:write" },
+          { "action": "grafana-oncall-app.schedules:write" },
+          { "action": "grafana-oncall-app.chatops:update-settings" },
+          { "action": "grafana-oncall-app.outgoing-webhooks:write" },
+          { "action": "grafana-oncall-app.user-settings:admin" },
+          { "action": "grafana-oncall-app.other-settings:write" }
+        ],
+        "hidden": true
```

Sorry, I removed the hidden field :/ Is it mandatory for you to hide this role?
```suggestion
        "hidden": true
```

---

noo this should be okay, I can express these grants a different way by just having a specific "Admin" role